### ISSUE TITLE
1.1.3 errors on large template file

### ISSUE THREAD

**Issue by clemsontiger** [edgardleal/require-vuejs#42]
I'm running into an issue where my vue template file is much longer than the others (x3) and the following line errors within the parse function:
```
var parse = function(text) {
        var doc = document.implementation.createHTMLDocument("");
        doc.body.innerHTML = text;
        var scriptElement = doc.getElementsByTagName("script")[0];
        var source = scriptElement.innerHTML;
        var css_result = css_parser.parseElement(doc);
        var template = template_parser.extractTemplate(doc, css_result);
        var functionString = css_result.functionString;
        if (!source || source.length < 10) {
            source = scriptElement.src;
            source = "define(['" + source + "'], function(comp) {\n comp.template = template;\n return comp;\n});\n";
        }
```
scriptElement is undefined. seems to work fine for my other vue templates, but this one it fails to find the script tag. This only happened once I upgraded to v 1.1.3. The template its erring on is 331kb. I'm guessing that a size limitation is happening and therefore it's truncating the template file not even loading the script block? Possibly the passed in "text" parameter?

**Comment by clemsontiger** [comment]
doc.body.innerHTML.length is 169776. If i take out alot of the template, it no longer runs into this issue, so it must have to be some type of size constraint issue

**Comment by clemsontiger** [comment]
Nevermind. Had nothing to do with the length of the file. I was able to narrow down an offending line of code that was causing the issue. I had  a <textarea> element that was closed off with just "/>" (invalid syntax, textarea is not self closing) instead of "</textarea"> once I fixed it, the issue went away.  While it has nothing to do with this repo, I wish somehow the underlying issue would have been more obvious, but oh well, I figured out without too much wasted time.
